README.md

```
@@ -19,62 +19,86 @@
 Distributed under the MIT License. See `LICENSE.txt` for more information.
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-
-
-
 <!-- CONTACT -->
+
 ## Contact
 
 TEAM keepintouch team.keepintouch.dev@gmail.com
```

Markdownlint

Bare URL used

(MD034, no-bare-urls)